src/gui/plugins/plotting/Plotting.cc

```diff
@@ -96,6 +97,11 @@ PlotComponent::PlotComponent(const std::string &_type,
   }
   else if (_type == "double")
     this->dataPtr->data["value"] = std::make_shared<PlotData>();
+  else if (_type == "Physics")
```

Comment:
I'm concerned that this won't scale well if we add a new type for each component type. For example, we plot `Vector` and `Pose`, instead of `Gravity` and `WorldPose`.

Could the 2 fields be sent independently as `double`s, so the plot doesn't need to know specifically about physics?

Reply:
I have been struggling a bit with the implementation of this since I'm not super familiar with the code base. I guess a similar feedback would apply to #482? (that I referenced for the implementation of the component_inspector / plotting tool parts) if that is the case I'll just keep an eye for the best fix there.

Reply:
> similar feedback would apply to #482?

Reply:
Ok, I looked closer at the code now. I see that `Plotting` needs to know about every single component that it's going to plot, because it needs to know how to access its data. So I think there isn't an easy way of doing what I suggested above, so just ignore it.